src/ui/popup/settings-page/enable-browser-theme.tsx

```diff
@@ -0,0 +1,20 @@
+import {m} from 'malevic';
+import CheckButton from '../check-button';
+import {ViewProps} from '../types';
+
+export default function EnableCustomBrowserTheme(props: ViewProps) {
+    function onEnableCustomBrowserTheme(checked: boolean) {
+        props.actions.changeSettings({changeBrowserTheme: !checked});
```

Comment:
As far as I understand it should be `props.actions.changeSettings({changeBrowserTheme: checked});` (without `!`).